components/content_settings/core/browser/brave_content_settings_pref_provider.cc

```diff
@@ -263,13 +262,16 @@ void BravePrefProvider::MigrateShieldsSettingsFromResourceIds() {
 }
 
 void BravePrefProvider::MigrateShieldsSettingsFromResourceIdsForOneType(
-    const std::string& preference_name,
+    const std::string& preference_path,
     const std::string& patterns_string,
     const base::Time& expiration,
     const base::Time& last_modified,
     SessionModel session_model,
     int setting) {
-  prefs::ScopedDictionaryPrefUpdate update(prefs_, preference_name);
+  // Non-supported preference paths should have been filtered out already.
+  DCHECK(prefs_->HasPrefPath(preference_path));
```

Comment:
shouldn't this be a `CHECK` with supporting text explanation since we know this will cause a crash?

Reply:
True, fixed